caramel/scripts/tool.py

```diff
@@ -79,22 +79,25 @@ def error_out(message):
 
 
 def print_list():
-    requests = models.CSR.valid()
-
-    def unsigned_last(csr):
-        return (not csr.certificates, csr.id)
-
-    requests.sort(key=unsigned_last)
```

Comment:
This part has been removed. I don't think that's desired.

Reply:
In my tests this part do nothing. I get the same[ output from my test-db](https://drive.google.com/drive/folders/1ALWe151_Tn2pjeiJCjxU4bos3lZ0OIOp?usp=sharing) with both the old and new code. But looking at it now, I've fixed that bug and will include it in the next pull request

Reply:
Actually, that was desired after testing both behaviours in our CA.